artwork_redirect/loggers.py

```diff
@@ -1,39 +0,0 @@
-import raven
-import raven.transport.threaded_requests
-from raven.handlers.logging import SentryHandler
-from raven.conf import setup_logging
-from werkzeug.exceptions import HTTPException
-import logging
-
-
-class MissingRavenClient(raven.Client):
-    """Raven client class that is used as a placeholder.
-    This is done to make sure that calls to functions in the client don't fail
-    even if the client is not initialized. Sentry server might be missing, but
-    we don't want to check if it actually exists in every place exception is
-    captured.
-    """
-    captureException = lambda self, *args, **kwargs: None
-    captureMessage = lambda self, *args, **kwargs: None
```

Comment:
This doesn't seem to be needed anymore either. I tried calling `sentry_sdk.capture_exception(Exception(''))` without running `sentry_sdk.init`, and it's a no-op.